Sources/Sentry/Profiling/SentryLaunchProfiling.m

```diff
@@ -20,12 +20,9 @@
 #    import "SentryTransactionContext.h"
 
 BOOL isTracingAppLaunch;
-SentryId *_Nullable appLaunchTraceId;
-NSObject *appLaunchTraceLock;
-uint64_t appLaunchSystemTime;
 NSString *const kSentryLaunchProfileConfigKeyTracesSampleRate = @"traces";
 NSString *const kSentryLaunchProfileConfigKeyProfilesSampleRate = @"profiles";
-SentryTracer *_Nullable launchTracer;
+static SentryTracer *_Nullable launchTracer;
```

Comment:
l: I believe that without an `NS_ASSUME_NONNULL_BEGIN` in the file, everything is assumed nullable. I understand that it does not affect the code, but I prefer clear, less cluttered code as much as possible.

Reply:
Actually I need to add those macros, so this _Nullable will need to stay as well